tests/lib/naming_test.py

```diff
@@ -169,5 +171,123 @@ def testParseServiceFile(self):
     filedefs._ParseFile(data, 'services')
     self.assertEqual(filedefs.GetService('HTTP'), ['80/tcp'])
 
+
+class Naming_Characterization_Tests(unittest.TestCase):
```

Comment:
Got it, np.  I was more concerned with getting the code into master, figuring that any duplication could be removed later.

They sort of are characterization tests, in that I wanted to explore and preserve the existing behaviour for a different implementation.  But that's semantic, and these can probably be removed.